webroot/src/pages/Home/Home.ts

```diff
@@ -41,7 +41,8 @@ class Home extends Vue {
 
         if (currentCompact) {
             const compactType = currentCompact.type;
-            const authType = this.$store.getters['user/highestPermissionAuthType']();
+            const authType = authStorage.getItem(tokens.staff.AUTH_TYPE)
+                || authStorage.getItem(tokens.licensee.AUTH_TYPE);
```

Comment:
Design question: Is there ever a time where `tokens.staff.AUTH_TYPE` is not of type staff, and `tokens.licensee.AUTH_TYPE` is not of type licensee?

Reply:
Actually yes, the way `auth_storage` is evaluated `tokens.staff.AUTH_TYPE` and `tokens.licensee.AUTH_TYPE` are actually stored in exactly the same place as each other in session / local storage. 

So if you set authStorage.setItem(tokens.licensee.AUTH_TYPE,  'licensee');  and then authStorage.getItem(tokens.staff.AUTH_TYPE) would equal 'licensee' because you are actually retrieving the same value you just set.)

Reply:
Hmm, got it. So then why do you have to call `getItem` on both variables here, given they refer to the same thing?

Reply:
I could do that get once, save it to a variable and then refer to that variable. I could add a comment too explaining that they are the same value.

It might also make more sense to move the auth_type value up out of the staff / licensee objects, so just authStorage.getItem(AUTH_TYPE) as this is more clear that this value is global?

Reply:
Yeah I think that makes more sense from my perspective, and would clarify the variable's role!